### PR TITLE
New package: TMSU-0.7.4

### PR DESCRIPTION
**Pull request by xaltsc**

### CODE REVIEW

File: srcpkgs/TMSU/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'TMSU'
+pkgname=TMSU
+version=0.7.4
+revision=1
+wrksrc="${pkgname}-${version}"
```
Comment (maxice8) [inline srcpkgs/TMSU/template]: default value, please remove

File: srcpkgs/TMSU/template [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+make_cmd="make -C ${wrksrc} MOUNT_INSTALL_DIR=/usr/bin"
+hostmakedepends="go git"
+depends="fuse sqlite>=3"
+short_desc="Tool for tagging files and accessing them through FUSE"
+maintainer="xaltsc <xaltsc@protonmail.ch>"
+license="GPL-3.0-or-later"
+homepage="https://tmsu.org"
+distfiles="https://github.com/oniony/${pkgname}/archive/v${version}.tar.gz"
+checksum=04e8a4c47532a4d8f673c3b3a921a47815c752b2ca1b564f6e1ed58a1b88a177
+nopie=yes
```
Comment (maxice8) [inline srcpkgs/TMSU/template]: no, make it respect our CFLAGS and build proper pie binaries.

Comment (xaltsc) [reply srcpkgs/TMSU/template]: I changed almost everything trying to build pie binaries.

File: srcpkgs/TMSU/update [hunk 3]
```diff
@@ -0,0 +1,2 @@
+site='https://github.com/oniony/TMSU/releases'
```
Comment (Chocimier) [inline srcpkgs/TMSU/update]: update file is not needed, defaults works here

File: srcpkgs/TMSU/template [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# Template file for 'TMSU'
+pkgname=TMSU
+version=0.7.4
+revision=1
+build_style=go
+go_import_path="github.com/oniony/${pkgname}"
+hostmakedepends="git"
+depends="fuse sqlite>=3"
```
Comment (Chocimier) [inline srcpkgs/TMSU/template]: sqlite is in version >=3 since the beginning, no need to specify that

File: srcpkgs/TMSU/template [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+short_desc="Tool for tagging files and accessing them through FUSE"
+maintainer="xaltsc <xaltsc@protonmail.ch>"
+license="GPL-3.0-or-later"
+homepage="https://tmsu.org"
+distfiles="https://github.com/oniony/${pkgname}/archive/v${version}.tar.gz"
```
Comment (Chocimier) [inline srcpkgs/TMSU/template]: Please don't use `${pkgname}`. It is always `TMSU`, so just use that.

**Comment by xaltsc** [comment]
Fixed everything and the build is now passing on all archs. I just forgot to include the license.

**Comment by Chocimier** [comment]
GPL does not need to be installed, see Manual.md.

**Comment by xaltsc** [comment]
Reverted.

**Pull request closed without merging**